### PR TITLE
FHAC 750: Create PerformanceTestTool profile which will add PerformanceTestTool jar if invoke

### PR DESCRIPTION
**Pull request by mhpnguyen**
Create PerformanceTestTool profile and the profile should add PerformanceTestTool jar if the profile is enabled.

### CODE REVIEW

**Comment by mhpnguyen** [comment]
@alameluchidambaram @christophermay07  Can you please review my first PR?

File: Product/Production/Deploy/pom.xml [hunk 1]
```diff
@@ -801,6 +801,21 @@
...
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>org.connectopensource</groupId>
+                    <artifactId>PerformanceTestTools</artifactId>
+                    <version>${project.parent.version}</version>
+                    <type>jar</type>
+                </dependency>               
+            </dependencies>           
+        </profile>
```
Comment (alameluchidambaram) [inline Product/Production/Deploy/pom.xml:819]: Can you format the xml?

File: Product/Production/pom.xml [hunk 2]
```diff
@@ -21,6 +21,7 @@
         <module>Adapters</module>
         <module>Gateway</module>
         <module>Deploy</module>
+		<module>../../../Plugins/PerformanceTestTools</module>
```
Comment (alameluchidambaram) [inline Product/Production/pom.xml]: Plugins module can be built separately. Do we need to include it in CONNECT build?

File: Product/Production/Deploy/pom.xml [hunk 3]
```diff
@@ -801,6 +801,21 @@
                 </plugins>
             </build>
         </profile>
+		<!-- Peform Test -->
```
Comment (christophermay07) [inline Product/Production/Deploy/pom.xml]: Can you change text to "Performance Test"?

**Comment by christophermay07** [comment]
add to whitelist

File: Product/Production/Deploy/pom.xml (hunk 1 above)
Comment (mhpnguyen) [inline Product/Production/Deploy/pom.xml:819]: Fix the format XML.

File: Product/Production/Deploy/pom.xml (hunk 3 above)
Comment (mhpnguyen) [inline Product/Production/Deploy/pom.xml]: Add "Performance Test" description.

File: Product/Production/Deploy/pom.xml [hunk 4]
```diff
@@ -802,6 +802,22 @@
...
+            <id>PerfTest</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>org.connectopensource</groupId>
+                    <artifactId>PerformanceTestTools</artifactId>
+                    <version>${project.parent.version}</version>
+                    <type>jar</type>
```
Comment (christophermay07) [inline Product/Production/Deploy/pom.xml]: Type "jar" is default for dependencies, this line should be removed.

**Comment by christophermay07** [comment]
Can you update the PR title (the commit with the semicolon changes is gone), and also add a brief PR description?

**Comment by christophermay07** [comment]
I've reviewed this PR, endorsing for merge without further comment.

**Comment by alameluchidambaram** [comment]
Reviewed PR and good with merge.

Going forward please put some information on description.

**Pull request merged**